app/services/get-export-csv.js

```diff
@@ -43,12 +42,14 @@ var getFields = function (organisationLevel, tab) {
         fieldNames = OM_OVERVIEW_FIELD_NAMES
       } else {
         childOrgForFieldName = getChildOrgForFieldName(organisationLevel)
-        fields = Object.assign([], ORG_OVERVIEW_FEILDS)
+        fields = Object.assign([], ORG_OVERVIEW_FIELDS)
         fieldNames = [childOrgForFieldName + 'Name', 'CapacityPercentage', 'CapacityPoints', 'ContractedHours', 'ReductionHours', 'TotalCases']
 
         if (organisationLevel === organisationUnitConstants.TEAM.name) {
           fields.push('gradeCode')
           fieldNames.push('GradeCode')
+          fields.unshift('lduCluster')
+          fieldNames.unshift('LDUCluster')
```

Comment:
According to the screenshots in the story, I think the team name should also be in the outputted CSV when at the team level. This may have changed though.

Reply:
Also, the names for the columns should have spaces between them. So `LDUCluster` should be `LDU Cluster`.